.github/workflows/promote-release.yml

```diff
@@ -0,0 +1,28 @@
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: version to promote to latest
+        type: string
+        required: true
+
+jobs:
+  promote:
+    runs-on: ubuntu-latest
+    environment: CLIS3BucketAndCloudfront
+    env:
+      CLOUDFRONT_DISTRIBUTION: ${{ secrets.CLOUDFRONT_DISTRIBUTION }}
+      HEROKU_S3_BUCKET: ${{ secrets.HEROKU_S3_BUCKET }}
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      AWS_EC2_METADATA_DISABLED: true
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js 16.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+          cache: yarn
+      - run: yarn --frozen-lockfile --network-timeout 1000000
+      - run: cd packages/cli
+      - run: oclif promote --deb --version=${{inputs.tag}}
```

Comment:
Where is `inputs` coming from? And should there be a space between the variable and the `{{`? So, like `${{ inputs.tag }}`

Reply:
I'm pretty sure that it needs to be added to an input field when selecting this workflow to run in the UI.
I don't know if whitespace matters, I'll add it. I copied most of this from here and modified: https://github.com/oclif/oclif/blob/main/.github/workflows/onRelease.yml

Reply:
Thank you!